imagetest/testworkflow.go

```diff
@@ -52,6 +52,35 @@ type TestWorkflow struct {
 	counter int
 }
 
+// Network defines the parameter which used by compute network.
+type Network struct {
+	Name         string
+	autoMode     bool
+	testWorkflow *TestWorkflow
+}
+
+// SubNetwork defines the parameter used by compute subnetwork.
+type SubNetwork struct {
+	Network        *Network
+	Name           string
+	primaryIPRange string
```

Comment:
just like the network struct includes a pointer to its test workflow, i think the subnet should or should include a pointer to its network